templates/.github/workflows/go-check.yml

```diff
@@ -18,6 +18,14 @@ jobs:
           go mod tidy
           diff go.mod go.mod.orig
           diff go.sum go.sum.orig
+      - name: go fmt
```

Comment:
nit: gofmt :) otherwise it might be surprising to the end user that Go files outside of Go packages are being formatted too.